flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/StringFunctionsITCase.java

```diff
@@ -25,12 +25,17 @@
 
 import static org.apache.flink.table.api.Expressions.$;
 import static org.apache.flink.table.api.Expressions.call;
+import static org.apache.flink.table.api.Expressions.lit;
 
 /** Test String functions correct behaviour. */
 class StringFunctionsITCase extends BuiltInFunctionTestBase {
 
     @Override
     Stream<TestSetSpec> getTestSetSpecs() {
+        return Stream.of(regexpExtractTestCases(), translateTestCases()).flatMap(s -> s);
```

Comment:
Maybe we can make this method more generic, because `concat` only works with two parameters, whereas we may need an arbitrary number of parameters, using `concat` just for two parameters specially seems unnecessary.